Assets/Scripts/UI/Camera/IN_GAME_MAIN_CAMERA.cs

```diff
@@ -244,11 +246,8 @@ public void setSpectorMode(bool val)
     {
         this.spectatorMode = val;
         GameObject.Find("MainCamera").GetComponent<SpectatorMovement>().disable = !val;
-
-        //TODO MouseLook
-        //GameObject.Find("MainCamera").GetComponent<MouseLook>().disable = !val;
+        GameObject.Find("MainCamera").GetComponent<Assets.Scripts.UI.Camera.MouseLook>().disable = !val;
```

Comment:
Import the namespace, and use MouseLook instead of Assets.Scripts.UI.Camera.MouseLook